test/NewsService.Data.UnitTests/NewsRepositoryTests.cs

```diff
@@ -48,10 +55,25 @@ public void SetUp()
                 AuthorId = _firstUserId,
                 SenderId = _firstUserId,
                 CreatedAt = DateTime.UtcNow,
+                DepartmentId = Guid.NewGuid(),
                 IsActive = true
             };
 
-            dbNewsToAdd = new DbNews
+            _editDbNews = new JsonPatchDocument<DbNews>();
+            _editDbNews.Operations.Add(new Operation<DbNews>("replace", "/Subject", "", _firstValue));
```

Comment:
Mb $”/{nameof(DbNews.Subject)}” would be better

Reply:
In other places too

Reply:
done